packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -168,6 +168,21 @@ enum SKProductDiscountPaymentMode {
   unspecified,
 }
 
+/// Dart wrapper around StoreKit's [SKProductDiscountType](https://developer.apple.com/documentation/storekit/skproductdiscounttype?language=objc)
+///
+/// This is used as a property in the [SKProductDiscountWrapper].
+// The values of the enum options are matching the [SKProductDiscountType]'s values. Should there be an update or addition
+// in the [SKProductDiscountType], this need to be updated to match.
```

Comment:
These comment lines are missing a forward slash (`/`). API documentation comments should start with three forward slashes (`///`).

Please also wrap documentation lines at 80 characters.

Reply:
Done.